x-pack/platform/packages/shared/security/api_key_management/src/components/token_field.tsx

```diff
@@ -36,10 +36,9 @@ export const TokenField: FunctionComponent<TokenFieldProps> = ({ value, ...props
         defaultMessage: 'Token',
       })}
       value={value}
-      style={{
+      css={{
```

Comment:
Replaced style with preferred css property. Removed `backgroundColor: 'transparent'` - no difference in render.